app/webpack/src/modules/proceedings.js

```diff
@@ -1,126 +1,122 @@
+  searchInputBox.addEventListener('keyup', (event) => {
     clearTimeout(typingTimer);
     typingTimer = setTimeout(doneTyping, doneTypingInterval);
   })
 
-  $("#proceeding-search-input").keydown(() => {
+  searchInputBox.addEventListener('keydown', () => {
     clearTimeout(typingTimer);
   })
 
-  $('#clear-proceeding-search').on("click", () => $("#proceeding-search-input").val("").trigger("keyup"))
+  document.querySelector('#clear-proceeding-search').addEventListener('click', () => {
+    searchInputBox.value = '';
+    hideProceeedingsItems();
+  })
 
   // TODO: remove this when the multiple proceedings feature flag is removed
-  $('li.proceeding-item').on('mouseover', function (e) { $(this).addClass('hover') })
-    .on('mouseout', function (e) { $(this).removeClass('hover') })
-    .on('click', function (e) { return submitForm(this) })
-    .on('keydown', function (e) {
-      if (e.which == 13) {
-        return submitForm(this)
-      }
+  const proceedingItems = document.querySelectorAll('.proceeding-item')
+  if (proceedingItems.length) {
+    proceedingItems.forEach((item) => {
+      item.addEventListener('mouseover', () => { item.classList.add('hover') })
+      item.addEventListener('mouseout', () => { item.classList.remove('hover') })
+      item.addEventListener('click', () => { return submitForm(item) })
+      item.addEventListener('keydown', (e) => {
+        if (e.which === 13) {
+          return submitForm(item);
+        }
+      })
     })
-}
-
-function search(proceedings_data) {
-  // For docs see: https://github.com/bvaughn/js-search
-  const search = new JsSearch.Search("code")
-  search.indexStrategy = new JsSearch.AllSubstringsIndexStrategy()
-
-  search.tokenizer = {
-    tokenize(text) {
-      return text.replace(/[^a-zA-Z\d]/gi, '').split()
-    }
   }
-
-  search.addIndex("meaning")
-  search.addIndex("description")
-  search.addIndex("category_law")
-  search.addIndex("matter")
-  search.addIndex("additional_search_terms")
-  search.addDocuments(proceedings_data)
-  return search
 }
 
-function searchCallback(event, search) {
-  $("#proceeding-list .proceeding-item").hide()
-  $(".no-proceeding-items").hide()
-
-  // Get user input and filter on it.
-  const inputText = $(event.target).val()
-  filterData(inputText, search)
-}
+// Find the existing hidden proceeding type items
+// If they are one of the search matches returned from the V1 api, remove the hidden class
+// and highlight the search terms in the item text
+function showResults (results, inputText) {
+  if (results.length > 0) {
+    const codes = results.map(obj => obj.code);
+    codes.forEach(code => {
+      // const element = $('#' + code)
+      const element = document.getElementById(code);
+
+      // We want to highlight anything text in <main> or <span> tags that
+      // matches the user's search criteria
+      const span = element.querySelector('span');
+      // remove h3 when the multiple proceedings feature flag is removed
```

Comment:
Might be worth adding a `TODO` to this comment to make it clearer that future action is required (similar to lines 43 and 109)

Reply:
will do!